frontend/components/BodyMarkdown.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div v-if="body.length" class="markdown">
+    <MarkdownHTML>
+      {{ body }}
+    </MarkdownHTML>
+  </div>
+</template>
```

Comment:
1. 

Small change, factoring out the `prme` body html that renders markdown (joined pr, and review) into a single component given that both slides and page views use this.